scripts/Snotel/get_snotelPoR_csv.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#
+# Download Snotel csv data from USDA for a site with end of day values in
+# metric units for the full period of record. Modified from get_snotel_csv.sh
+# Default state is Colorado, update as needed.
+# Data includes: Depth, SWE, Density, Wind Speed, Air Temperature, Precipitation
+#
+# More information is in the header of the generated csv file.
+#
+# Arguments:
+#  1: Snotel Site ID
+#
+# Outputs: CSV file
+#   Name: [downloaddate]_PoR_site[SiteID].csv
+#
+# Example:
+#  get_snotelPoR_csv.sh 112
+
+HOST="https://wcc.sc.egov.usda.gov/reportGenerator/view_csv/customMultiTimeSeriesGroupByStationReport,metric/daily/end_of_period/"
+COLUMNS="/POR_BEGIN,POR_END/SNWD::value,WTEQ::value,SNDN::value,WSPDV::value,TOBS::value,PRCP::value"
+STATE="CO"
```

Comment:
As a side note - You could make this parameter optional and assign a default value with:
```
STATE="${VARIABLE:-CO}"
```

This is what's called [parameter expansion in bash](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html)